doc/sphinx/source/esmvaldiag/observations.rst

```diff
@@ -0,0 +1,265 @@
+************************************************************
+Contributing a CMORizing script for an observational dataset
+************************************************************
+
+ESMValTool is able to process observational data sets, however, since many
+features of this version are based on the `Iris library 
+<https://scitools.org.uk/iris/docs/latest/>`_ that uses the `CF conventions
+<http://cfconventions.org/>`_, the format of the data sets has to follow very
+strict CMOR rules. If the original observational data set is not in the right
+CMOR format, it has to be CMORized before the ESMValTool preprocessor can
+work with it. The following steps are necessary to prepare an observational
+data set for the use in ESMValTool.
+
+| `1. Check if your variable is CMOR standard`_
+| `2. Edit your configuration file`_
+| `3. Store your dataset in the right place`_
+| `4. Create a cmorizer for the dataset`_
+| `5. Run the cmorizing script`_
+| `6. Naming convention of the observational data files`_
+| `7. Test the cmorized dataset`_
+
+
+1. Check if your variable is CMOR standard
+==========================================
+
+Most variables are defined in the CMIP data request and can be found in the
+CMOR tables in the folder ``/esmvalcore/cmor/tables/cmip5/Tables/``,
```

Comment:
Note that this will be moved to the ESMValTool Core repostitory, a link there might be nice. It would also be good to mention that this is just a copy of the PCMDI repo

Reply:
I added the link to the ESMValTool Core repository and also added a sentence about PCMDI.